templates/App/Controller/FormAction.html.twig

```diff
@@ -1,5 +1,10 @@
 {% extends 'base.html.twig' %}
 
+{% types {
+    dto: 'App\\Dto\\RegisterFormDto',
```

Comment:
Double `\` won't be necessary with Twig4 @ruudk, right?

Reply:
Correct.